src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs

```diff
@@ -20,15 +21,18 @@ namespace Microsoft.PowerFx
     public sealed class PowerFxConfig
     {
         private bool _isLocked;
-        private readonly HashSet<TexlFunction> _extraFunctions = new HashSet<TexlFunction>();
+        private readonly HashSet<TexlFunction> _extraFunctions = new HashSet<TexlFunction>();
+        private readonly HashSet<IGlobalSymbol> _globalSymbols = new HashSet<IGlobalSymbol>();
```

Comment:
>IGlobalSymbol [](http://example.com/codeflow?start=33&length=13)

Dictionary<string, IGlobalSymbol> ? 


HashSet<IGlobalSymbol> implies that it implements GetHashCode / Equals (which are not part of the interface).

Reply:
I don't see a need